### ISSUE TITLE
fix a11y issue for all social links

### ISSUE THREAD

**Issue by kirandash** [bgwebagency/redesigner.io#13]
Fix any of the following:

Element does not have text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"

![image](https://github.com/bgwebagency/redesigner.io/assets/13310363/6952adcd-c060-48f7-a9ac-975e733bc074)
